src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -2817,6 +2829,22 @@
     text: az bicep decompile --file {json_template_file} --force
 """
 
+helps['bicep decompile-params'] = """
+type: command
+short-summary: Attempts to decompile a parameters .json file to .bicepparam.
```

Comment:
Please use the first person voice in the help message

Reply:
got it! 👍